scripts/run_lint_python.sh

```diff
@@ -28,9 +31,6 @@ display_result $? 1 "Flake 8 code style check"
 find . -type f -name "*.py" | xargs pylint --reports=n --output-format=colorized --rcfile=.pylintrc -j 0
 display_result $? 1 "Pylint linting check"
 
-isort --check .
```

Comment:
Is it worth removing this from the packages, as well as the others that ruff replaces?

Reply:
I think I removed everything now, `pep8` is used by `pylint-mccabe` and pylint stays for now.